source/uri-options/uri-options.rst

```diff
@@ -339,7 +339,8 @@ pertaining to URI options apply here.
      - positive number
      - defined in the `Connection Pooling spec`_
      - required for drivers with connection pools, with exceptions described in the `Connection Pooling spec`_
-     - The maximum number of clients or connections able to be created by a pool at a given time
+     - If specified, requests MUST time out after spending waitQueueTimeoutMS or
```

Comment:
nit: The other timeout options don't reference themselves in their definitions. For example, the description for `socketTimeoutMS` is
> Amount of time spent attempting to send or receive on a socket before timing out

Maybe change this one to `Amount of time spent attempting to check out a connection from a server's connection pool before timing out`?

Reply:
Makes sense, I applied your suggestion.